packages/cli/src/commands/setup/auth/__tests__/addAuthConfigToApp.test.js

```diff
@@ -80,3 +84,23 @@ describe('Should add config lines to App.{js,tsx}', () => {
     await addConfigToApp(nhostData.config, false)
   })
 })
+
+describe('Should add config lines when RedwoodApolloProvider has props', () => {
+  it('Matches Auth0 Snapshot', async () => {
+    mockWebAppPath =
```

Comment:
I think this should be reset between tests. Otherwise this might affect tests that are run after this test that don't explicitly mock the path. You can add `mockWebAppPath = ''` to the `beforeEach` above where we restore all mocks.